vxyowsup/whatsapp.py

```diff
@@ -154,8 +154,8 @@ def onMessage(self, messageProtocolEntity):
             messageProtocolEntity.getParticipant())
         self.toLower(receipt)
 
-        print "You have received a message, and thusly sent a receipt"
-        print "You are now sending a reply"
+        log.debug('You have received a message, and thusly sent a receipt')
+        # log.debug("You are now sending a reply")
         # self.send_to_human('iiii', from_address + '@s.whatsapp.net', "this transport's gone rogue")
```

Comment:
It would just be for debugging. This code has been useful to you for
debugging, so presumably it will be useful to other people too (e.g. me),
so it would be nice to keep it around in a way that doesn't involve having
to edit the source code to activate it or specify which MSISDN to send to.